android/titanium/build.gradle

```diff
@@ -278,7 +278,7 @@ dependencies {
 	implementation fileTree(dir: 'lib', include: ['*.jar'])
 
 	// CameraX
-  def camerax_version = '1.1.0-beta03'
+  def camerax_version = '1.1.0-rc02'
```

Comment:
Nice! Finally in RC!

Reply:
yes! And only some bug fixes so hopefully it's final soon :crossed_fingers: